mods/ra/maps/allies-05a/allies05a.lua

```diff
@@ -202,7 +203,7 @@ FreeTanya = function()
 		Media.PlaySpeechNotification(greece, "TargetFreed")
 	end
 
-	if not SpecialCameras then
+	if not SpecialCameras and PrisonCamera then
```

Comment:
Isn't `if PrisonCamera then` sufficient?

Reply:
Actually not, `PrisonCamera` gets removed in two different places, depending on if `SpecialCameras` is true or not (and we don't want to attempt to remove it twice).